service/controller/machinepool/handler/migration/testdata/case-2-New-MachinePool-fully-created_MachinePool.golden

```diff
@@ -0,0 +1,91 @@
+apiVersion: cluster.x-k8s.io/v1beta1
```

Comment:
New MachinePool CR that got created and the migration is fully complete (Metadata, Spec and Status are updated). New MachinePool migration handler does not do anything, so here we see that the new MachinePool CR is not changed.